canonical/ubuntu.com
# WD-16840 Rebranded /landscape/pricing page

## Done

- Rebranded page
- Rename pricing to compare
- Add redirect from /landscape/pricing to /landscape/compare

## QA

- View the site  in your web browser at: https://ubuntu-com-14581.demos.haus/landscape/pricing
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check that it matches the following:
   - [Design](https://www.figma.com/design/btsxPLFKGPWDlbQ4rjR4jZ/25.04-Landscape-Rebranding?node-id=4015-2351&t=2riHUPsatn9YCZDu-0)
   - [CopyDoc](https://docs.google.com/document/d/1pESmRniuiaDHkbzY_60GlOJdCiH2pZ9erI9KV49eDyM/edit?tab=t.0#)

## Issue / Card

Fixes #[WD-16840](https://warthogs.atlassian.net/browse/WD-16840)

[WD-16840]: https://warthogs.atlassian.net/browse/WD-16840?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
